### ISSUE TITLE
new feature: RVN+ZIL dual mining

### ISSUE THREAD

**Issue by geopapai** [todxx/teamredminer#247]
Hi, please add RVN+ZIL functionality in addition to ETH+ZIL.

**Comment by junl2004** [comment]
+1

**Comment by Crayphish** [comment]
+1, nice feature that is implemented in nanominer where the ZIL DAG is held held in VRAM as well so that the switch can be done quickly.

**Comment by junl2004** [comment]
> +1, nice feature that is implemented in nanominer where the ZIL DAG is held held in VRAM as well so that the switch can be done quickly.

 i tried nanominer , it has ~ 25% lower RVN HR than TRM

**Comment by Haikv** [comment]
+ for rvn+zil

**Comment by snnln** [comment]
+1

**Comment by TURKOGLUKUTAY** [comment]
+1

**Comment by NoanKa** [comment]
+1

**Comment by hugochief** [comment]
+1

**Comment by arando** [comment]
Please!!!

**Comment by isbrodie1** [comment]
> > +1, nice feature that is implemented in nanominer where the ZIL DAG is held held in VRAM as well so that the switch can be done quickly.
> 
> i tried nanominer , it has ~ 25% lower RVN HR than TRM

Yes, also tried Nanominer as a work around any my Hash dropped  from 12.5 to 9.
Its working on TMR but not for 4GB cards it seems
